### PR TITLE
PT-164691690 Allow mutual close after solo close/slash

### PR DESCRIPTION
**Pull request by grzegorz225**
No changes in aeternity/protocol repo are needed.

### CODE REVIEW

File: apps/aechannel/src/aesc_fsm.erl [hunk 1]
```diff
@@ -1061,6 +1062,14 @@ awaiting_signature(cast, {?SIGNED, settle_tx, SignedTx} = Msg,
                 settle_signed(SignedTx, Updates, D1)
         end, D);
 
+% Timeouts
```
Comment (uwiger) [inline apps/aechannel/src/aesc_fsm.erl]: A minor quibble, but use two %% at the beginning of a line. The Emacs indentation mode will try to indent a single %

File: apps/aechannel/src/aesc_fsm.erl [hunk 2]
```diff
@@ -2496,6 +2535,11 @@ adjust_ttl(TTL) when is_integer(TTL), TTL >= 0 ->
 cur_height() ->
     aec_headers:height(aec_chain:top_header()).
 
+is_fork_active(ProtocolVSN) ->
+    CurHeight = max(cur_height(), ?MINIMUM_DEPTH),
+    %% Enable a new fork only after MINIMUM_DEPTH generations in order to avoid fork changes
+    ProtocolVSN == aec_hard_forks:protocol_effective_at_height(CurHeight - ?MINIMUM_DEPTH).
```
Comment (uwiger) [inline apps/aechannel/src/aesc_fsm.erl]:
```suggestion
    ProtocolVSN == aec_hard_forks:protocol_effective_at_height(max(cur_height() - ?MINIMUM_DEPTH, aec_block_genesis:height())).
```

File: apps/aechannel/src/aesc_fsm.erl [hunk 3]
```diff
@@ -2496,6 +2535,11 @@ adjust_ttl(TTL) when is_integer(TTL), TTL >= 0 ->
 cur_height() ->
     aec_headers:height(aec_chain:top_header()).
 
+is_fork_active(ProtocolVSN) ->
+    CurHeight = max(cur_height(), ?MINIMUM_DEPTH),
```
Comment (uwiger) [inline apps/aechannel/src/aesc_fsm.erl]:
```suggestion
```

Comment (uwiger) [reply apps/aechannel/src/aesc_fsm.erl]: It feels counter-intuitive to call this `CurHeight` when it possibly isn't.

File: apps/aechannel/src/aesc_fsm.erl (hunk 1 above)
Comment (grzegorz225) [reply apps/aechannel/src/aesc_fsm.erl]: Ok, good to know. BTW a quick grep of the codebase ```grep -r -e "^% " apps | wc -l ``` shows that there are 66 occurrences of a single % at the beginning of a line.

**Pull request merged**